etl/steps/data/garden/ember/2024-05-08/yearly_electricity.py

```diff
@@ -356,15 +426,44 @@ def run(dest_dir: str) -> None:
         latest_year = tables[table_name].reset_index()["year"].max()
```

Comment:
That's the most likely place where this issue happens. It's good to check in at least one place, but not in all places (because some aggregates may not suffer from this issue).